### PR TITLE
Remove unused feature commands

### PR DESCRIPTION
**Pull request by CrendKing**
## PR Summary

Issue report is https://github.com/PowerShell/vscode-powershell/issues/3586. These two commands are unused except a `dispose()` call (I suppose these are the remnants of some historical refactoring?) In the generated js file, the declarations are gone, but the calls to `dispose()` remain, which causes the extension's `dispose()` to fail.

I suppose alternatively we could tweak how js files are generated and not optimize out unused variables, but that could have way more footprint than removing them. However, if these variables are still used in some way, you guy decide how to proceed.

## PR Checklist

Note: Tick the boxes below that apply to this pull request by putting an `x` between the square brackets.
Please mark anything not applicable to this PR `NA`.

- [x] PR has a meaningful title
- [x] Summarized changes
- [`NA`] PR has tests
- [x] This PR is ready to merge and is not work in progress
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready

### CODE REVIEW

File: src/features/DebugSession.ts [hunk 1]
```diff
@@ -51,10 +50,6 @@ export class DebugSessionFeature extends LanguageClientConsumer
         return new vscode.DebugAdapterInlineImplementation(debugAdapter);
     }
 
-    public dispose() {
-        this.command.dispose();
-    }
```
Comment (andschwa) [inline src/features/DebugSession.ts:56]: Thanks for the contribution! Looks like due to the abstraction this class is inheriting, we'll need to leave `public dispose()` implemented (but empty):

```
src/features/DebugSession.ts(24,14): error TS2515: Non-abstract class 'DebugSessionFeature' does not implement inherited abstract member 'dispose' from class 'LanguageClientConsumer'.
```

**Review by andschwa — APPROVED** [review]
Also please sign the CLA! I'd love to take this PR today. Thanks so much!

File: src/features/DebugSession.ts (hunk 1 above)
Comment (CrendKing) [reply src/features/DebugSession.ts:56]: Done

Comment (andschwa) [reply src/features/DebugSession.ts:56]: Well dang, that triggered:

> ERROR: 53:22  no-empty  block is empty

Are you able to build this locally? TSC should complain in the same way as CI here.

Comment (CrendKing) [reply src/features/DebugSession.ts:56]: You are right. I could either temporarily disable the tslint or add harmless variable:
```
/* tslint:disable:no-empty */
public dispose() {
}
/* tslint:enable:no-empty */
```
public dispose() {
    const noop = undefined;
}
```

Comment (andschwa) [reply src/features/DebugSession.ts:56]: Sweet!

Comment (CrendKing) [reply src/features/DebugSession.ts:56]: Done. CI passed.

**Comment by andschwa** [comment]
Thanks so much @CrendKing!

**Pull request merged**